Description/it_it.md

```diff
@@ -0,0 +1,89 @@
+# Italiano (It)
+
+Un modpack di minecraft creato per la performance e miglioramenti grafici.
+
+[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
+
+### Performance
+
+Fabulously Optimized è di solito più veloce della OptiFine grazie a [mod moderne per la performance][1] come Sodium, Lithium e tante altre!
+
+### Parità della OptiFine
+
+Fabulously Optimized [supporta quasi tutte le features della OptiFine][2], incluse le shaders, textures connesse e zoom. [Puoi anche avere il tuo mantello gratuito!][3]
+
+### Miglioramenti Minori
+
+Fabulously Optimized [include dei piccoli miglioramenti][4] che non rovinano l'esperienza vanilla, come la cronologia della chat estesa, fullscreen in modalità finestra, fixes per i bug più comuni e strumenti per trattarli.
+
+### Funziona Ovunque!
+
+Fabulously Optimized non ha mod di content, quindi funziona su tutti i server. Il modpack [può essere usato su 5 launcher, ed è hostato su 3 siti][6] ed [è tradotto in diverse lingue][7], incluso l'italiano!
```

Comment:
[Mod Menu Helper has not been translated](https://fabulously-optimized.gitbook.io/modpack/readme/language-support#translating-mod-menu-helper) to Italian yet, so the sentence ending is incorrect :P

Reply:
oh right